libyul/backends/evm/ConstantOptimiser.h

```diff
@@ -58,7 +58,7 @@ class ConstantOptimiser: public ASTModifier
 	struct Representation
 	{
 		std::unique_ptr<Expression> expression;
-		size_t cost = size_t(-1);
+		bigint cost = size_t(-1);
```

Comment:
What does this mean?

Reply:
I guess treating `2**64 - 1` as infinity.

Reply:
Sure, but is this mechanism still working?

Reply:
I actually think the initial value is never used anywhere. I'll leave it as the old one.

Reply:
but shouldn't it rather be zero then?

Reply:
Updated it to 0